src/actions/guides/getting-started/installing.cr

```diff
@@ -77,7 +77,7 @@ class Guides::GettingStarted::Installing < GuideAction
     * cd into the newly cloned directory
     * Check out latest [released version](https://github.com/luckyframework/lucky_cli/releases) `git checkout #{LuckyCliVersion.current_tag}`
     * Run `shards install`
-    * Run `crystal build src/lucky.cr`
+    * Run `crystal build --release src/lucky.cr`
```

Comment:
@paulcsmith I thought this should be here, but what's your thoughts on this? Not sure if there's a reason to have this in a non-release build.

Reply:
The main reason was that it takes forever to build in release mode and the CLI does so little that the performance optimizations aren't noticeable. So I think we can keep it as it was